packages/inventory-general-info/src/BiosCard.js

```diff
@@ -4,14 +4,18 @@ import { connect } from 'react-redux';
 import LoadingCard from './LoadingCard';
 import { generalMapper } from './dataMapper';
 import { biosSelector } from './selectors';
+import { DateFormat } from '@redhat-cloud-services/frontend-components';
 
 const BiosCard = ({ bios, detailLoaded, handleClick }) => (<LoadingCard
     title="BIOS"
     isLoading={ !detailLoaded }
     items={ [
         { title: 'Vendor', value: bios.vendor },
         { title: 'Version', value: bios.version },
-        { title: 'Release date', value: bios.releaseDate },
+        { title: 'Release date', value: (DateFormat ?
```

Comment:
Just curious: is this check done because the application using this package does not need to have the frontend-components package?

Reply:
Not exactly. The date format component is not in an older version of frontend-components package and it could just blow up.

Reply:
Should not it be set as a peer dependency then?

Reply:
Not really, it is set as dependency, so it'll be installed all the times. However if the app has older version of it this could blow up.